packages/babel-plugin-expo-vector-icons/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org/"
+  },
+  "name": "babel-plugin-expo-vector-icons",
+  "version": "1.0.0",
+  "description": "Babel plugin for eliminating unused fonts by redirecting all @expo/vector-icons",
+  "main": "index.js",
+  "devDependencies": {
+    "babel-core": "^6.26.3",
```

Comment:
Having two versions of Babel in the repo is more likely to cause issues somewhere down the line. Update this to use Babel 7.